src/test/java/org/jabref/logic/xmp/XmpUtilWriterTest.java

```diff
@@ -134,6 +155,136 @@ void testWriteMultipleBibEntries(@TempDir Path tempDir) throws IOException, Tran
         assertEquals(3, entryList.size());
     }
 
+    /**
+     * Test for writing XMP metadata to PDF file and export without enclosing braces.
+     */
+    @Test
+    void testWriteAndExportXmpWithoutEnclosingBraces(@TempDir Path tempDir) throws Exception {
+        // turn filter on
+        when(xmpPreferences.shouldEnableEnclosingBracketsFilter()).thenReturn(true);
+        // some initialization
+        BibDatabaseMode bibDatabaseMode = BibDatabaseMode.BIBTEX;
+        BibEntryTypesManager bibEntryTypesManager = new BibEntryTypesManager();
+        FieldWriterPreferences fieldWriterPreferences = new FieldWriterPreferences(true, List.of(StandardField.MONTH), new FieldContentFormatterPreferences());
+        BibDatabaseContext databaseContext = new BibDatabaseContext();
+        BibDatabase database = databaseContext.getDatabase();
+
+        FilePreferences filePreferences = mock(FilePreferences.class);
+        when(filePreferences.getUser()).thenReturn(tempDir.toAbsolutePath().toString());
+        when(filePreferences.shouldStoreFilesRelativeToBibFile()).thenReturn(false);
+
+        Path pdfFile = tempDir.resolve("existing.pdf").toAbsolutePath();
+        // read a bib entry from the tests before
+        BibEntry entry = vapnik2000;
+        // link file to entry
+        LinkedFile linkedFile = createDefaultLinkedFile("existing.pdf", tempDir);
+        entry.setFiles(Arrays.asList(linkedFile));
+        // insert entry to database
+        database.insertEntry(entry);
+
+        String expectBibTeX = "@Book{vapnik2000,\n" +
+                "  author    = Vladimir N. Vapnik,\n" +
+                "  publisher = Springer Science + Business Media,\n" +
+                "  title     = The Nature of Statistical Learning Theory,\n" +
+                "  year      = 2000,\n" +
+                "  month     = may,\n" +
+                "  coverage  = coverageField,\n" +
+                "  doi       = 10.1007/978-1-4757-3264-1,\n" +
+                "  language  = English, Japanese,\n" +
+                "  owner     = Ich,\n" +
+                "  rights    = Right To X,\n" +
+                "  source    = JabRef,\n" +
+                "}";
```

Comment:
This is wrong. This is wrong BibTeX.

The author of the issue meant another formatter: https://docs.jabref.org/finding-sorting-and-cleaning-entries/saveactions#unprotect-terms

See https://github.com/JabRef/jabref/issues/8452#issuecomment-1108980819 for more details